src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR;
+    private static final int DEFAULT_VALUE;
+    private static final int NUMBER_OF_FIGURES;
+    private Random random;
+    private ColorSupplier colorSupplier;
+
+    static {
+        DEFAULT_COLOR = Color.WHITE.name();
+        DEFAULT_VALUE = 10;
+        NUMBER_OF_FIGURES = 5;
+    }
```

Comment:
the constant is a constant because it never changes, and therefore it can not be null or 0 and then be changed to these values after the initialization, and static blocks of initialization are in general a bad practice
```suggestion
```